syntax = "proto3";

package keto.proto;

/**
 * This is an example of a protobuf message.
 */
message CheckForAccount {
    uint32 version = 1;
    bytes account_hash = 2;
    bool found = 3;
}

message AccountInfo {
    uint32 version = 1;
    bytes account_hash = 2;
    bytes management_account_hash = 3;
    string graph_name = 4;
}



